brainfuck/commands_test.go

```diff
@@ -0,0 +1,51 @@
+package brainfuck
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestLoop_execute(t *testing.T) {
```

Comment:
pls test each command. Assert that the current pointer and memory is correct after you run `Execute`

Reply:
Fixed